sample/app/build.gradle

```diff
@@ -25,6 +25,9 @@ buildProperties {
     releaseSigning {
         file rootProject.file('properties/releaseSigning.properties')
     }
+    env {
```

Comment:
I think not just in the sample, we should also update `README` to include this change.